CMakeLists.txt

```diff
@@ -228,17 +228,9 @@ add_glob_target(
   TOUCH_STRATEGY SINGLE)
 add_dependencies(lintlua-luacheck lua-dev-deps)
 
-add_glob_target(
-  TARGET lintlua-stylua
-  COMMAND ${STYLUA_PRG}
-  FLAGS --color=always --check
-  GLOB_DIRS runtime/
-  GLOB_PAT *.lua
-  EXCLUDE
-    /runtime/lua/vim/re.lua
-    /runtime/lua/vim/_meta/.*
-  TOUCH_STRATEGY SINGLE)
-
+add_custom_target(lintlua-stylua
```

Comment:
worth a brief comment?
```suggestion
# No GLOB_PAT so that .styluaignore works. https://github.com/JohnnyMorganz/StyLua/issues/751
add_custom_target(lintlua-stylua
```